CHANGES.md

```diff
@@ -1,3 +1,20 @@
+## edge-19.10.4
+
+This edge release adds dashboard UX enhancements, Open Census tracing
+for the control plane components, and improves the speed of the CLI.
+
+* CLI
+  * Made `linkerd install --ignore-cluster` and `--skip-checks` faster
+  * Fixed a bug causing `linkerd upgrade` to fail when used with 
+  `--from-manifest`
+* Controller
+  * Added `--trace-collector` and `--sampling-probability` flags to the 
+    control plane components to enabled the export of Open Census tracing data
```

Comment:
Since `--trace-collector` is an endpoint, can we say something like:

> ... to the control plane components which specifies the spans collector URL.